base/cvd/cuttlefish/host/commands/cvd/cli/command_request.cpp

```diff
@@ -39,6 +40,14 @@ const selector::SelectorOptions& CommandRequest::Selectors() const {
 
 const cvd_common::Envs& CommandRequest::Env() const { return env_; }
 
+std::string CommandRequest::Subcommand() const {
+  return ParseInvocation(*this).command;
```

Comment:
This has the potential of parsing multiple times in case of multiple calls to either of these functions. Can the call to `ParseInvocation` happen in the constructor?

Reply:
Implemented in [48aa26a](https://github.com/google/android-cuttlefish/pull/816/commits/48aa26a5582041dbb51c1036d566c1a0c0990dbc).